### ISSUE TITLE
Trouble on OSX when building this with objc_meta and d-metal-bindings

### ISSUE THREAD

**Issue by wilsonk** [MrcSnm/d-avaudioengine-binding#1]
I am using the "something".ns coding style on OSX (arm64) for some bindings. My code is 'nothrow @nogc' but version 1.0.15 of objc_meta seems to fail and 1.1.0 seems to be necessary to use 'nothrow @nogc'? 

However, if I use 1.1.0 (which I want to) then d-avaudioengine requires 1.0.15? Anyways, I modified the dub.json file to account for this (if there is no major reason that avaudioengine wouldn't work with objc_meta 1.1.0?), and also split out the framework for ios and osx. It seems as though AVFoundation (which includes AVAudioEngine) is needed on OSX, instead of just AVAudioEngine like ios. Here it is:

```
{
	"authors": [
		"Hipreme"
	],
	"configurations": [
		{
			"lflags-ios": [
				"-framework",
				"AVAudioEngine"
			],
			"lflags-osx": [
				"-framework",
				"AVFoundation"
			],
			"name": "macOS",
			"platforms": [
				"osx"
			],
			"targetType": "library"
		},
		{
			"name": "test_generation",
			"targetType": "library"
		}
	],
	"copyright": "Copyright © 2023, Hipreme",
	"dependencies": {
		"objc_meta": "~>1.1.0"
	},
	"description": "AVAudioEngine binding for D.",
	"importPaths": [
		"source/"
	],
	"license": "MIT",
	"name": "avaudioengine",
	"sourcePaths": [
		"source/"
	],
	"version": "1.0.15"
}
```

**Comment by MrcSnm** [comment]
I'll issue a new release using the newest objc meta

**Comment by MrcSnm** [comment]
Fixed in version 1.1.0 - https://code.dlang.org/packages/avaudioengine
